src/main/java/uk/ac/ebi/atlas/experimentpage/markergenes/MarkerGenesDao.java

```diff
@@ -11,9 +10,9 @@
 @Repository
 public class MarkerGenesDao {
     private final NamedParameterJdbcTemplate namedParameterJdbcTemplate;
-    private static final String CELL_GROUP_TYPE = "inferred cell type";
+	private static final String CELL_GROUP_TYPE = "inferred cell type - ontology labels";
```

Comment:
Shouldn't this be `inferred cell type _-_ ontology labels`? As this is how they are stored in Solr

Reply:
We are querying the database, where they are stored with spaces. Confusing enough... It’d be good to try to normalise this though. Can you add a story to the backlog, please? This needs to be discussed with data production, too.

Reply:
I will add the story!